components/31-molecules/cta-block/README.md

```diff
@@ -1,17 +1,49 @@
-# CTA links
+# Call to action
 
-A CTA link has a certain behavior.
+## When to use this component
 
-* It should always have a width based on the page container of 7/12. This means
-  the container needs to make sure CTA links can never be bigger then 7/12 of itself.
-* It always has a max-width 40 rems on desktop.
-* For a mobile layout a CTA link is always full width.
+Use the call to action component on a page:
 
-Examples of paragraphs are defined in paragraph variants:
+* To guide users to the next step to reach a certain goal related to the page.
+* To present specific contact information to get help or to get more information from a specitic service to reach a certain goal.
+* To promote a certain action for the user to take.
 
-* <a href="{{path './cta-link--website'}}">CTA link website</a>
-* <a href="{{path './cta-link--document'}}">CTA link document</a>
-* <a href="{{path './cta-link--form-fill'}}">CTA link form fill</a>
-* <a href="{{path './cta-link--email'}}">CTA link email</a>
-* <a href="{{path './cta-link--phone'}}">CTA link phone</a>
-* <a href="{{path './cta-link--location'}}">CTA link location</a>
+## When not to use this component
+
+Do not use the call to action component for other uses than the uses listed above.
+
+## How it works
+
+The call to action component consists of the following parts:
+
+* A title representing the call to action (required)
+* A spot illustration (required and fixed, depending on the type of the primary action)
+* A short text explaining the call to action (optional)
+* A primary action (required)
+* A secondary action (optional)
+
+The primary action can be one of the following types:
+
+* A link to a web page (internal or external)
+* A link to a document
+* A link to an e-mail address (mailto-link)
+* A link to a telephone number (tel-link)
+
+The optional secondary action can be one of the following types:
```

Comment:
1/ The secondary action could be indeed also an e-mail address or a telephone number. I cannot see why we would not allow that in the Style Guide.

The only problem I can think of is that one could show two e-mail address or two telephone number without a clear difference. This can be confusing for users. What do you think?

2/ Also a good idea to mention that in the text there cannot be links.

Reply:
@delrueba The documentation has been updated.

Reply:
1/ It's not included in the allowed types in your list ;)

2/ :+1: 

3/ I like this list style :)